Documents/SCA Learning Resources/ProductDesign/resources.md

```diff
@@ -0,0 +1,6 @@
+# Product Design Resources
+
```

Comment:
It would be nice if we could get a little description of what to here. You can check the backend or frontend resource for an example